src/utils/web3.js

```diff
@@ -88,8 +121,49 @@ export const bscRpcsMN = [
   'https://bsc-dataseed1.ninicoin.io/',
 ]
 
-// ABI
-export const ERC20_ABI = ERC20.abi
+/**
+ * Trigger change between Addresses
+ * @param {string} net - 'mainnet' or 'testnet'
+ * @returns {Object} Relevant list of addresses
+ */
+export const changeAddresses = (_network) => {
```

Comment:
why you include underscore in the argument network? usually underscore is only use to indicate in javascript that one method of a class is private

Reply:
In react it doesn't matter, has no privacy implications, I only did the underscore because I wanted to keep the same function name but it was already being used by a const at the time that I added it.
Good pickup though, I'll ensure that const is gone now and change it to 'network' 👍